astrocut/cube_cut.py

```diff
@@ -403,7 +404,7 @@ def _get_cutout(self, transposed_cube, threads: Union[int, Literal["auto"]] = 1,
         -------
         response :  `numpy.array`, `numpy.array`, `numpy.array`
```

Comment:
should this line change as well? the second item is either a numpy array or None.

some type annotations on these functions would also help a lot. i think it would be valuable to add mypy at some point to this project and to type annotate as much as we can.